neuralprophet/utils.py

```diff
@@ -987,3 +987,153 @@ def configure_trainer(
     # config["replace_sampler_ddp"] = False
 
     return pl.Trainer(**config), checkpoint_callback
+
+
+def unpack_sliced_tensor(
+    sliced_tensor,
+    n_lags,
+    n_forecasts,
+    max_lags,
+    feature_indices,
+    config_lagged_regressors,
+    config_seasonality,
+):
+    sliced_tensor = sliced_tensor.detach().clone()
+    inputs = OrderedDict()
+    if max_lags > 0:
+        # Unpack time feature (time doesn't need further slicing)
+        start_idx, end_idx = feature_indices["time"]
```

Comment:
Can we create an analogous time_offset dict for each feature? 
Storing time_offset and time_steps